Adding di, tri, and tetra Ubiquitin structures

hjjvandam opened the pull request:
For the DFT capabilities we need larger test cases than a single Ubiquitin protein to be able to get sensible timings. Fortunately poly-Ubiquitin proteins are naturally occurring with distinct biological functions. In addition these systems are very similar to Ubiquitin making them very suitable as more extended systems. From this class of proteins this PR adds LYS-48 linked di-, tri-, and tetra-Ubiquitin.

hjjvandam committed:
Adding tests for reference codes.

hjjvandam committed:
Fixed the file names.

hjjvandam committed:
Updating file names.

hjjvandam committed:
Fixing this structure.

hjjvandam committed:
Adding more structures.

hjjvandam committed:
Removing crystal waters, Zn and SO4 ions.

hjjvandam committed:
PyBel script to optimize 1AAR protein structure.

hjjvandam committed:
Got this to work (somewhat).

hjjvandam committed:
First unsuccessful attempt.

hjjvandam committed:
Removing superfluous atoms that resulted from pdb4amber disconnecting the chains.

hjjvandam committed:
Merge branch 'master' of ssh://github.com/hjjvandam/NWX_TA

hjjvandam committed:
Fixing jobscripts, etc.

hjjvandam committed:
Merge branch 'master' of ssh://github.com/hjjvandam/NWX_TA

hjjvandam committed:
First working script.

hjjvandam committed:
First optimization result. Still nowhere close.

hjjvandam committed:
Making a cyclic structure out of 1AAR does not work.

Basically you have to connect the loose end of one Ubiquitin to
LYS-48 of the other. Then you need to re-optimize the structure.
However, using OpenBabel rotamer searches does not get you anywhere
close. Trying an energy minimization does not work because the bond
that needs to be fixed is massively overstretched generating huge
forces that pull the system into totally unphysical territory. Also
the system does not have enough symmetry to try and use that to
construct a sensible system from just the Ubiquitin chain that is
already linked. Beyond these simple attempts to fix this solving this
problem becomes a research problem in its own right that we do not have
the time for. So we will just use the available 1AAR structure and
kick the rest to the curb.

hjjvandam committed:
Fix some layout stuff.

hjjvandam committed:
Replacing 1aar_h.pdb with the properly connected structure.

hjjvandam committed:
Fix out of date statement.

hjjvandam committed:
Merge branch 'NWChemEx-Project:master' into master

hjjvandam committed:
Providing XYZ format for NWChem input.

hjjvandam committed:
Merge branch 'master' of ssh://github.com/hjjvandam/NWX_TA

hjjvandam committed:
Adding XYZ format for NWChem input.

hjjvandam committed:
Adding input files and jobscripts.

hjjvandam committed:
Structure after 100,000 timesteps of dynamics, one more check to come.

hjjvandam committed:
Merge branch 'master' of ssh://github.com/hjjvandam/NWX_TA

hjjvandam committed:
Comments on 1aar_cyclic.pdb.

hjjvandam committed:
Adding XYZ version of the 1aar_cyclic structure.

hjjvandam committed:
Files for Cori

hjjvandam committed:
Omit github token.

hjjvandam committed:
Updating structures with MD minimalized ones.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/NWChemEx-Project/NWX_TA?pullRequest=3) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/NWChemEx-Project/NWX_TA?pullRequest=3) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/NWChemEx-Project/NWX_TA?pullRequest=3) it.</sub>

hjjvandam committed:
Zeolite cluster with 1 active site.

This cluster has been used as the basis for calculations within
the SRP-HPC projects with Prof. Eugene Mananga. In particular
Aissata Diop, a student on the team, helped build this fragment.

hjjvandam committed:
Zeolite fragment with 2-propanol near active site

This is the Zeolite fragment with 2-propanol added that has been
used in the SRP-HPC project.
